vomsimporter.py

```diff
@@ -497,10 +523,12 @@ def import_voms_user(self, voms_user):
             logging.info("User has email override, disable HR email sync")
             self.add_skip_email_synch_label(iam_user)
 
+        iam_group_names = set()
         for f in voms_user['fqans']:
             logging.info("Importing %s membership in VOMS FQAN: %s",
                          iam_user_str, f)
             iam_group_name = fqan2iam_group_name(f)
+            iam_group_names.add(iam_group_name)
             iam_group = self.find_group_by_name(iam_group_name)
```

Comment:
Here you make a call to IAM for each user and for each group, even if the groups don't change. Wouldn't it be possible to store all the IAM groups like you do for the VOMS groups?

Reply:
On line 531? I'm just adding string to the set()

Reply:
Next line, sorry. I mean the call `self.find_group_by_name(iam_group_name)`

Reply:
I have added some caching in a new branch, see #11. Please have a look.